src/07_slices.py

```diff
@@ -1,7 +1,7 @@
 """
-Python exposes a terse and intuitive syntax for performing 
```

Comment:
Perfect! 🎉  List comprehensions and slice notations will come in handy a lot throughout CS.